File: 20200423/code.py
def solution():
  n = int(input())
  arr = [int(input()) for i in range(n)]
  arr.sort(reverse=True)
  sums = []
  for i, v in enumerate(arr):
    sums.append(v * (i+1))
  
  return max(sums)


print(solution())

File: 20200423/code.js
function solution(arr) {
  arr.sort((a, b) => b - a);
  max = 0;
  for (let i=0; i<arr.length; i++) {
    if(max < arr[i] * (i+1)){
      max = arr[i] * (i+1)
    } else {
      return max
    }
  }
  return max
}

console.log(solution([10,20,30]));
console.log(solution([10,15]));